src/v2/view-builder/internals/BaseForm.js

```diff
@@ -58,7 +58,7 @@ export default Form.extend({
 
   saveForm(model) {
     //remove any existing warnings or messages before saving form
-    this.$el.find('.o-form-error-container').empty();
+    this.$el.find('.o-form-error-container .infobox-error').remove();
```

Comment:
nit: since we're only clearing errors now, we should update the comment above to reflect that now. 

Related question, with this change we're not going to be clearing warning messages anymore at the base form level, just ensuring that this won't cause any regressions. I don't think it will since warning messages were recently introduced but wanted to bring to your attention to be sure.

Reply:
Good point. Will explore some other way of doing this